src/renderer.ts

```diff
@@ -204,10 +213,24 @@ class Renderer extends EventEmitter<RendererEvents> {
     return [div, shadow]
   }
 
+  /** Wavesurfer itself calls this method. Do not call it manually. */
   setOptions(options: WaveSurferOptions) {
+    if (this.options.container !== options.container) {
+      const newParent = this.parentFromOptionsContainer(options.container)
+      this.parent.removeChild(this.container)
+      newParent.appendChild(this.container)
+
+      this.parent = newParent
+      this.emit('containerRemount')
```

Comment:
The name of the event isn’t consistent with other events. None of them are in camelCase.

Reply:
Removed the event.